Jenkinsfile

```diff
@@ -21,7 +21,7 @@
 // - Periodically cleanup the old versions on local workers
 //
 ci_lint = "tvmai/ci-lint:v0.50"
-ci_gpu = "tvmai/ci-gpu:v0.51"
+ci_gpu = "tvmai/ci-gpu:v0.52"
```

Comment:
keep this as v0.51 since that is already the latest image

Reply:
Sure